modules/Bio/Otter/Server/UserSpecies.pm

```diff
@@ -0,0 +1,59 @@
+package Bio::Otter::Server::UserSpecies;
+
+use strict;
+use warnings;
+use DBI;
+use Data::Dumper;
+sub species_group{
+
+ # MySQL database configurations
+ my $dsn = "DBI:mysql:otter_registration";
+ my $username = 'root';
+ my $password = 'sorciers';
+
```

Comment:
DB settings should be extracted in settings file, besides it will improve code flexibility, with current code we can't share share it since it has hardcoded db settings.

Reply:
This will be removed.